docs/howto/ejb-remote-security/index.adoc

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
So is this a HOWTO? I originally thought that we'd add this to the `ejb-remote` fraction docs, but I agree that a HOWTO is better!